test/d2l-input-checkbox.html

```diff
@@ -481,18 +476,19 @@
 		describe('set', function() {
 
 			beforeEach(function() {
-				elem = fixture('tabindex');
+				elem = fixture('basic');
+				elem.tabbable = false;
 			});
 
-			it('should set "tabindex" to "-1"', function() {
-				expect(elem.getAttribute('tab-index')).to.equal('-1');
+			it('should set "tabbable" to "false"', function() {
+				expect(elem.getAttribute('tabbable')).to.equal(null);
 				expect(elem.$$('input').tabIndex).to.equal(-1);
 			});
 
-			it('should reflect "tabindex" attribute change to property and input', function() {
-				elem.setAttribute('tab-index', '1');
-				expect(elem.getAttribute('tab-index')).to.equal('1');
-				expect(elem.$$('input').tabIndex).to.equal(1);
+			it('should reflect "tabbable" attribute change to property and input', function() {
+				elem.setAttribute('tabbable', false);
```

Comment:
Boolean attributes should be set to themselves for "true" and removed entirely for "false". So `elem.setAttribute('tabbable', 'tabbable')` and `elem.removeAttribute('tabbable')` respectively.